src/cython/cython/alpha_complex.pyx

```diff
@@ -5,19 +5,19 @@ from libcpp.string cimport string
 from libcpp cimport bool
 import os
 
-"""This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+""" This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
     See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
     Author(s):       Vincent Rouvreau
 
-   Copyright (C) 2016 Inria
+    Copyright (C) 2016 Inria
 
-   Modification(s):
-     - YYYY/MM Author: Description of the modification
+    Modification(s):
+      - YYYY/MM Author: Description of the modification
 """
 
 __author__ = "Vincent Rouvreau"
 __copyright__ = "Copyright (C) 2016 Inria"
-__license__ = "GPL v3"
+__license__ = "MIT"
```

Comment:
I think it might be better to keep GPL there. The license of the file is written above, it is MIT. But the complete package including the dependency on CGAL is GPL. And if someone queries gudhi.AlphaComplex.__license__ (or however that shows up in python), it would be better if they got the information: GPL.